### ISSUE TITLE
Allow empty string values for 'aria-current' attribute

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#994]
ARIA 1.1 spec for aria-current states
> If the attribute is not present or its value is an empty string or undefined, the default value of false applies and the aria-current state MUST NOT be exposed by user agents or assistive technologies.

Consider allowing empty strings for ```aria-current```

<pre><code>axe-core version: 3.0.3
axe-Coconut version: 3.4.0-alpha.2</code></pre>

**Comment by WilcoFiers** [comment]
Sounds easy enough :)

**Comment by marcysutton** [comment]
We should test this to make sure it is accessibility supported.

**Comment by iamrafan** [comment]
AT test results for ```aria-current```

  | Browser/AT | aria-current=""   Announced as | aria-current="false"   Announced as | aria-current="true"   Announced as
-- | -- | -- | -- | --
1 | Edge/Narrator | Not announced | Not announced | Not announced
2 | Edge/JAWS | Not announced | Not announced | Not announced
3 | Edge/NVDA | Not announced | Not announced | "Label"   current
4 | Chrome/NVDA | Not announced | Not announced | "Label"   current
5 | Chrome/JAWS | Not announced | Not announced | "Label"   current
6 | IE/JAWS | Not announced | Not announced | "Label"   current
7 | IE/NVDA | Not announced | Not announced | "Label"   current
8 | Safari/voiceover/OS   X | Not announced | Not announced | Not announced
9 | Firefox/NVDA | Not announced | Not announced | "Label"   current

**Comment by WilcoFiers** [comment]
Looking into a fix on this, made wonder if we should just add an "allowEmpty" property to our aria attributes.

**Comment by WilcoFiers** [comment]
Apologies, closed it by accident. Thanks for the catch :-).

**Comment by iamrafan** [comment]
Np, thanks for reopening the issue
